core/modules/views/includes/view.inc

```diff
@@ -624,9 +625,9 @@ class view {
 
     $this->relationships_fixed = TRUE;
 
-    // Go through all of our handler types and test them to see if they
-    // are missing relationships. Missing relationships can cause fatally
-    // broken Views.
+    // Go through all of our handler types, and test them to see if they are
```

Comment:
No comma needed here.

Reply:
These two variations would make sense to me (I went with the first):

>Go through all of our handler types, and test them to see if they are missing relationships.

>Go through all of our handler types and test them, to see if they are missing relationships.

Reply:
I'll concede this one in order to get this committed :slightly_smiling_face: